Tests/FunctionalTests/PluginTests.swift

```diff
@@ -34,6 +34,15 @@ class PluginTests: XCTestCase {
         }
     }
 
+    func testUseOfBuildToolPluginTargetNoPreBuildCommandsByExecutableInSamePackage() throws {
```

Comment:
This name is too long and difficult to read. Would we be able to come up with a more concise name that still uniquely identifies the test?

Reply:
Done @MaxDesiatov